docs/specs/validation/content-validation.yml

```diff
@@ -135,7 +135,7 @@ outputs:
   .errors.log: |
     {"message_severity":"suggestion","code":"sensitive-language","message":"Term 'master' is sensitive and should not be used in content or code.","file":"a/b/c/file.md","line":1,"end_line":1,"column":0,"end_column":0}
     {"message_severity":"suggestion","code":"sensitive-language","message":"Term 'master' is sensitive and should not be used in content or code.","file":"file1.md","line":1,"end_line":1,"column":0,"end_column":0}
-    {"message_severity":"warning","code":"rule-override-invalid","message":"Validation rule 'sensitive-language' is not overridable, so overrides in docfx.yml/docfx.json will be ignored.","file":"docfx.yml","line":4,"end_line":4,"column":5,"end_column":5}
+    {"message_severity":"warning","code":"rule-override-invalid","message":"Validation rule 'sensitive-language' is not overridable, so overrides in docfx.yml/docfx.json will be ignored.","line":0,"end_line":0,"column":0,"end_column":0}
```

Comment:
it is because when do code refactor, we change `SourceInfo<CustomRule>` to `CustomRule` in `CustomRuleProvider` class to clean up code. @yufeih should know the context.